analytics/analytics_test.go

```diff
@@ -0,0 +1,59 @@
+package analytics
+
+import (
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestAnalyticsSubmitPostsToHeap(t *testing.T) {
+	assert := assert.New(t)
+	called := false
```

Comment:
(nitpick) Use the zero value, as Rob Pike says in his proverbs:
`var called bool` is the same.

(Just my personal preference, nothing wrong with your code).

Reply:
I prefer your suggested syntax too. Updating now.